### PR TITLE
Fix/add broken channels back

### PR DESCRIPTION
**Pull request by link8k**

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
IMO I understand for Ugarit, but for the 2 others... Are they better in terms of server stability / definition (we could, then, benefit from it by adding a definition mark next to the channel name (SD/HD))

**Comment by link8k** [comment]
iq.m3u8 has new links. The player picks the best stream for the network.

File: channels/sy.m3u [hunk 1]
```diff
@@ -27,3 +27,5 @@ https://svs.itworkscdn.net/syriatvlive/syriatv.smil/playlist.m3u8
 http://82.137.248.16:1935/SEdu/stream10112020/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/6KUtkKz.png" group-title="News",Syrian News Channel
 http://82.137.248.16:1935/Snews/stream10112020/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/4x2uwpx.png" group-title="",Ugarit TV [not 24/7]
+http://82.137.248.16:1935/Ugarit/stream10112020/playlist.m3u8
```
Comment (themak1985) [inline channels/sy.m3u:31]: What did you use to test this, I can't get it to work - is it a part time channel?

**Review by themak1985 — COMMENTED** [review]
@link8k whats the deal with urgarit - I can't get it to work

File: channels/sy.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/sy.m3u:31]: Just tested now, it answers. I think it's not a 24/7 channel.

Comment (link8k) [reply channels/sy.m3u:31]: It works for half the day (whilst the channel is broadcasting).

Thanks @LaneSh4d0w

**Pull request merged**